lib/src/cli/format_command.dart

```diff
@@ -189,9 +189,12 @@ class FormatCommand extends Command<int> {
       }
     }
 
-    var pageWidth = int.tryParse(argResults['line-length'] as String) ??
-        usageException('--line-length must be an integer, was '
-            '"${argResults['line-length']}".');
+    int? pageWidth;
+    if (argResults.wasParsed('line-length')) {
+      pageWidth = int.tryParse(argResults['line-length'] as String) ??
+          usageException('--line-length must be an integer, was '
+              '"${argResults['line-length']}".');
+    }
 
     var indent = int.tryParse(argResults['indent'] as String) ??
         usageException('--indent must be an integer, was '
```

Comment:
Should the be an `if (line-length < 1)` test. (Or `< _minimumSaneWidth`? I'd be fine with refusing to work for widths less than twice the longest keyword in the language, or something.)

Reply:
I don't know. I don't think it's much of a problem in practice.

Reply:
I guess someone actually saying `--line-length=-4` isn't going to get much worse results than someone saying `--line-length=1`. And if it crashes the formatter, I guess they asked for it! Ship it!